### ISSUE TITLE
Uncaught InvalidCharacterError: Failed to execute 'add' on 'DOMTokenList': [...] - Information and Concise File Included

### ISSUE THREAD

**Issue by steventhanna** [atom-community/linter#800]
I have encountered an error while working in a very basic Java file.  
This issue is related to issue #795 .
```java
import java.io.*;
import java.util.*;

public class Detect {

  private long byteCount = 0;
  private Hashtable fileList = new Hashtable();

  public Detect() {

  }

  public boolean isDirectory(File file) {
    if(file.isDirectory()) {
      return true;
    } else {
      return false;
    }
  }

  public void walkFileTree(File file) {
    if(isDirectory(file)) {
      File[] files = file.list();
      for(int i = 0; i < files.length; i++) {
        if(isDirectory(files[i])) {
          walkFileTree(files[i]);
        } else {
          String name = files[i].getName();
          if(fileList.get(name) == null) {
            fileList.put(name, new Double(1));
          } else {

          }
        }
      }
    }
  }
}
```
The code above triggered the error: `Uncaught InvalidCharacterError: Failed to execute 'add' on 'DOMTokenList': The token provided ('error:-incompatible types') contains HTML space characters, which are not valid in tokens.`

More specifically, as soon as I started to implement the hashtable `fileList.put(name)`, the error was thrown.  The Javac linter did throw a warning: `[unchecked] unchecked call to put(K,V) as a member of the raw type Hashtable`.  It is possible that this error message is getting a space character to aid in line wrapping.  Although I am running Atom on a Mac in full-screen mode, I was able to reproduce the error outside of full-screen mode.

Another explanation is that there might be a hidden HTML space character somewhere in my code. However this is unlikely because none of the code was copy and pasted from another source.  All code was created in editor.

Both the javac linter and the regular linter were updated prior to the error being thrown.

Oddly enough, when I deleted the code causing the problem, the errors were still thrown, even though the code that was supposed to throw the error in the linter has been deleted.

**Atom Version**: 1.0.3
**System**: Mac OS X 10.10.4
**Thrown From**: [linter](https://github.com/atom-community/linter) package, v1.3.0

### Stack Trace

Uncaught InvalidCharacterError: Failed to execute 'add' on 'DOMTokenList': The token provided ('warning:-[rawtypes] found raw type') contains HTML space characters, which are not valid in tokens.

```
At /Users/steventhanna/.atom/packages/linter/lib/views/message.coffee:30

Error: Failed to execute 'add' on 'DOMTokenList': The token provided ('warning:-[rawtypes] found raw type') contains HTML space characters, which are not valid in tokens.
  at Error (native)
  at Function.Message.renderRibbon (/Users/steventhanna/.atom/packages/linter/lib/views/message.coffee:30:18)
  at linter-message.Message.attachedCallback (/Users/steventhanna/.atom/packages/linter/lib/views/message.coffee:5:26)
  at linter-panel.<anonymous> (/Users/steventhanna/.atom/packages/linter/lib/views/bottom-panel.js:49:12)
  at Array.forEach (native)
  at linter-panel.updateMessages (/Users/steventhanna/.atom/packages/linter/lib/views/bottom-panel.js:48:14)
  at LinterViews.renderPanelMessages (/Users/steventhanna/.atom/packages/linter/lib/linter-views.coffee:107:12)
  at LinterViews.render (/Users/steventhanna/.atom/packages/linter/lib/linter-views.coffee:42:6)
  at /Users/steventhanna/.atom/packages/linter/lib/linter-plus.coffee:29:14
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at MessageRegistry.updatePublic (/Users/steventhanna/.atom/packages/linter/lib/message-registry.coffee:55:16)
  at /Users/steventhanna/.atom/packages/linter/lib/message-registry.coffee:57:31

```

### Commands

```
     -2:49.7.0 editor:newline (atom-text-editor.editor.is-focused)
  4x -2:46.6.0 core:save (atom-text-editor.editor.is-focused)
     -2:35.4.0 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
     -1:23.2.0 whitespace:remove-trailing-whitespace (atom-text-editor.editor.is-focused)
     -1:19.2.0 core:save (atom-text-editor.editor.is-focused)
     -0:48.7.0 encoding-selector:show (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:45.2.0 core:save (atom-text-editor.editor.is-focused)
     -0:39.6.0 encoding-selector:show (atom-text-editor.editor.is-focused)
     -0:34.7.0 core:save (atom-text-editor.editor.is-focused)
     -0:21.3.0 core:select-all (atom-text-editor.editor.is-focused)
     -0:10.5.0 core:copy (atom-text-editor.editor.is-focused)
     -0:10.1.0 core:cut (atom-text-editor.editor.is-focused)
  2x -0:08.7.0 core:save (atom-text-editor.editor.is-focused)
     -0:02.3.0 core:select-all (atom-text-editor.editor.is-focused)
     -0:01.9.0 core:paste (atom-text-editor.editor.is-focused)
     -0:01.4.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "close-tags",
      "vim-mode"
    ]
  },
  "linter": {}
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.28.5
autoclose-html, v0.18.0
bracket-matcher, v0.76.0
color-picker, v2.0.4
editor-stats, v0.17.0
language-ejs, v0.1.0
linter, v1.3.0
linter-javac, v1.0.0
minimap, v4.10.0
wakatime, v4.0.10

# Dev
No dev packages
```

**Comment by steelbrain** [comment]
Dupe of #795
